ome/omero-iviewer
# trello requested fixes for region edit constraints and cursor appearance

see: https://trello.com/c/HYnRRhE8/243-review-first-round


This PR addresses the following:

1) A 10px upper limit on the stroke width was removed
2) A 1px lower limit on the stroke width was changed to 0px so that any filled geometry (lines exl.!!) can have no stroke
3) The font size min bound was lowered to 1, the upper limit of 1000 removed
4) There were supposed browser interpretations of the 'col-resize' cursor style as 4 way arrows such as  IE which was replaced by 'ew-resize' which seems to show the horizontal 2 way arrow always.


How to Test: 

Choose an image with regions (on cowfish) and try setting the stroke width and font size to values that were not allowed prior to this PR. For the mouse cursor (4) you need to move the mouse over the the edge of the left hand or right hand panel and the 2 arrow cursor should appear. 
Note: the cursor when hovering over the collapse icon will look different.
